### PR TITLE
added consumer offsets docs

### PR DESCRIPTION
**Pull request by galibey**

### CODE REVIEW

File: content/docs/concepts/producer-consumer.md [hunk 1]
```diff
@@ -27,3 +27,152 @@ not expire after being consumed.
...
+
+### Consumer Offsets
+**Consumer Offset** is the [offset] value assigned to the specified consumer name. Upon consuming more records from the topic a consumer updates the offset by performing periodic commit/flush operations. Later, this offset can be retrieved from the **Fluvio** cluster to continue reading from the last record.  
+
+**Consumer Offsets** are persistent and durable, surviving Fluvio cluster restarts and upgrades. Essentially, Fluvio provides the same storage guarantees for Consumer Offsets as it does for the data stored in topics.  
+
+The offset value is maintained separately for each topic partition. Once created, it remains stored until explicitly deleted.  
+Users can delete the offset either programmatically through their application code or via the Fluvio CLI.
+
+#### Auto commits
```
Comment (ajhunyady) [inline content/docs/concepts/producer-consumer.md:39]: Should be "Commit Strategy"

File: content/docs/concepts/producer-consumer.md [hunk 2]
```diff
@@ -27,3 +27,152 @@ not expire after being consumed.
...
+               .build()?,
+       )
+       .await?;
+   while let Some(Ok(record)) = stream.next().await {
+       println!("{}", String::from_utf8_lossy(record.as_ref()));
+   }
+   Ok(())
+}
+```
+[offset]: {{< ref "/docs/concepts/offsets" >}}
```
Comment (ajhunyady) [inline content/docs/concepts/producer-consumer.md:178]: Can we add a section for connectors?

File: content/docs/concepts/producer-consumer.md (hunk 1 above)
Comment (galibey) [reply content/docs/concepts/producer-consumer.md]: Done

File: content/docs/concepts/producer-consumer.md (hunk 2 above)
Comment (galibey) [reply content/docs/concepts/producer-consumer.md:204]: Added

**Review by ajhunyady — APPROVED** [review]

**Pull request merged**